src/components/Article.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable react/no-unescaped-entities */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import ArticleBody from '../views/ArticleBody';
+/*
+ * @todo - Import comments component here
+ */
+
+export const ArticleContext = React.createContext({ article: {} });
+
+class Article extends React.Component {
+  static propTypes = {
+    match: PropTypes.object.isRequired,
+  }
+  constructor(props) {
+    super(props);
+    this.state = {
+      article: {},
+      bookmarked: false,
+    };
+  }
+
+  componentDidMount = async () => {
+    const { match } = this.props;
+    const { slug } = match.params;
+    const result = await axios.get(`${process.env.API_BASE_URL}/articles/${slug}`);
+    this.setState({ article: result.data.data });
+  }
+
+  bookmarkArticle = () => {
```

Comment:
#### Comment
What about if we tried this rather? 
```
this.setState{(
     bookmarked: !this.state.bookmarked
)}
```

Let me know your thoughts?

Reply:
This is actually much better than using the tenary operator there. Thank you for suggesting this. You just opened my eyes to it.